wp-seo-main.php

```diff
@@ -374,6 +382,46 @@ function wpseo_init() {
 	}
 }
 
+/**
+ * Detects if we can invalidate the opcache and does so.
+ *
+ * @return bool
+ */
+function wpseo_maybe_invalidate_opcache() {
+	require_once ABSPATH . 'wp-admin/includes/file.php';
+
+	if ( ! function_exists( 'wp_opcache_invalidate' ) || ! function_exists( 'wp_opcache_invalidate_directory' ) ) {
+		return false;
+	}
+
+	$access_type = get_filesystem_method();
+	if ( $access_type !== 'direct' ) {
+		return false;
+	}
+
+	// Prevent any output requesting credentials, even though we should be safe in 'direct' mode.
+	ob_start();
+	$credentials = request_filesystem_credentials( admin_url() );
```

Comment:
The `request_filesystem_credentials()` function is being executed on `plugins_loaded`, however the documentation states that this can only be used _AFTER_ the `wp_loaded` hook, and only on admin requests.